### ISSUE TITLE
[BUG] Need keyboard-accessible way to resize columns

### ISSUE THREAD

**Issue by ExtAnimal** [bryntum/support#13433]
When focused on a grid column header it should be possible to resize that column.

### Plan

Add a "Resize column" item to the column header menu. The text must be localized.

Activating that option should show a dialog similar to below to enable the new size to be specified:

<img width="597" height="400" alt="Image" src="https://github.com/user-attachments/assets/4b757e77-9c9b-45cb-ac9e-e9915d79c110" />
